Nodejs/Product/Npm/SPI/PackageJson.cs

```diff
@@ -153,6 +155,17 @@ internal class PackageJson : IPackageJson {
             }
         }
 
+        private void InitRequiredBy() {
+            try {
+                RequiredBy = (_package["_requiredBy"] as IEnumerable<JToken> ?? Enumerable.Empty<JToken>()).Values<string>();

```

Comment:
OK. Looks like this line addresses the above comment (and the RequiredBy enumerator shouldn't be null).